external/deep-object-reid/torchreid_tasks/utils.py

```diff
@@ -256,8 +256,8 @@ def generate_label_schema(not_empty_labels: List[LabelEntity], multilabel: bool
 
 
 @check_input_parameters_type()
-def get_multihead_class_info(label_schema: LabelSchemaEntity):
-    all_groups = label_schema.get_groups(include_empty=False)
+def get_multihead_class_info(label_schema: LabelSchemaEntity, with_empty=False):
```

Comment:
`with_empty` makes me little confused. How about to use `include_empty`? I'm okay with `label_schema.get_labels(include_empty=include_empty)`